extensions/dicom-seg/src/components/SegmentationSettings.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect } from 'react';
+import { Range } from '@ohif/ui';
+
+import './SegmentationSettings.css';
+
+const SegmentationSettings = ({ configuration, onBack, onChange }) => {
+  const [state, setState] = useState({ ...configuration });
```

Comment:
Oh man. What in the world is in configuration? What if they pass something we don't expect? Suddenly reactive.

Reply:
Added prop types for it, maybe we need to add validations/ value boundaries?